app/models/alchemy/node.rb

```diff
@@ -22,7 +22,11 @@ class Node < BaseRecord
     # Either the value is stored in the database
     # or, if attached, the values comes from a page.
     def name
-      read_attribute(:name).presence || page&.name
+      if root?
+        Alchemy.t(read_attribute(:name), scope: :menu_names)
+      else
+        read_attribute(:name).presence || page&.name
+      end
```

Comment:
We should fix that in a future PR. The name should always be translated in the DB.